sklearn/linear_model/_base.py

```diff
@@ -46,6 +47,57 @@
 # intercept oscillation.
 
 
+# FIXME remove function in 0.28
+def _deprecate_normalize(normalize, default):
+    if normalize == 'deprecated':
+        _normalize = default
+    else:
+        _normalize = normalize
+
+    if default:
+        assert parse_version(sklearn.__version__) < parse_version('0.26'), (
```

Comment:
I don't think we should write the code for 0.26 yet. I think we should just write the deprecation needed for 0.24, and write instructions in the comments about what should be done in 0.26 and 0.28 instead.

Reply:
Ok. Like that is better @NicolasHug ?